other/tools/run_protoc.sh

```diff
@@ -24,7 +24,11 @@ command -v protoc-gen-mavsdk > /dev/null || {
 }
 
 function snake_case_to_camel_case {
-    echo $1 | sed -r 's/(^|_)([a-z])/\U\2/g'
+    if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Where does `$OSTYPE` come from?

Reply:
It is from one of the first search hits on "how to determine the os type in bash scripts" 😀 

It seems to be defined by default.